tests/src/App/ElementMap.cpp

```diff
@@ -3,16 +3,198 @@
 #include "gtest/gtest.h"
 
 #include "App/ElementMap.h"
+#include <App/Application.h>
```

Comment:
Is there any reason this and the above headers are imported with different formats?